packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacket/index.tsx

```diff
@@ -70,42 +74,53 @@ export const useStyles = makeStyles()((theme) => {
         },
         from: {
             fontSize: '14px',
-            color: '#FFFFFF',
-            fontWeight: 'bold',
+            color: theme.palette.common.white,
+            alignSelf: 'end',
+            fontWeight: 500,
             [`@media (max-width: ${theme.breakpoints.values.sm}px)`]: {
                 fontSize: 14,
                 right: 12,
                 bottom: 8,
             },
         },
         label: {
+            width: 76,
+            height: 27,
+            display: 'flex',
+            justifyContent: 'center',
+            fontSize: 12,
+            alignItems: 'center',
             borderRadius: theme.spacing(1),
-            padding: theme.spacing(0.2, 1),
-            background: 'rgba(0, 0, 0, 0.2)',
+            backgroundColor: parseColor(theme.palette.common.black).setAlpha(0.5).toString(),
             textTransform: 'capitalize',
             position: 'absolute',
             right: 12,
-            top: 8,
+            top: 12,
         },
         words: {
-            color: '#FAD85A',
-            fontSize: 20,
-            whiteSpace: 'pre',
+            display: '-webkit-box',
+            '-webkit-line-clamp': '2',
+            '-webkit-box-orient': 'vertical',
```

Comment:
Use `WebKitLineClamp` and `WebkitBoxOrient` instead, because React will complain about these attributes.

<img width="891" alt="image" src="https://user-images.githubusercontent.com/1141198/217703957-30cf0cdf-6f94-4cf0-a136-7309e58acb17.png">
<img width="947" alt="image" src="https://user-images.githubusercontent.com/1141198/217703980-febdbe88-4b3a-444f-874a-efe01f48a9f8.png">